lib/widgets/payment_request_dialog.dart

```diff
@@ -55,7 +60,13 @@ class PaymentRequestDialogState extends State<PaymentRequestDialog>
   String _amountToPayStr;
 
   bool _inProgress = false;
-  bool _isInit = false;
+
+  static const double _titleHeight = 48.0;
+  static const double _actionsHeight = 64.0;
+  double _dialogContentHeight;
```

Comment:
why do we need to define `_dialogContentHeight` as class variable? we only need to calculate it when needed on paymentfulfilled.

Reply:
https://github.com/breez/breezmobile/pull/94#discussion_r283110876